cmd/seid/cmd/root.go

```diff
@@ -302,6 +305,21 @@ func newApp(
 		baseapp.SetSnapshotDirectory(cast.ToString(appOpts.Get(server.FlagStateSyncSnapshotDir))),
 		baseapp.SetOccEnabled(cast.ToBool(appOpts.Get(baseapp.FlagOccEnabled))),
 	)
+
+	// Start migration if --migrate flag is set
+	if cast.ToBool(appOpts.Get("migrate-iavl")) {
+		go func() {
+			homeDir := cast.ToString(appOpts.Get(flags.FlagHome))
+			stateStore := app.GetStateStore()
+			latestVersion := rootmulti.GetLatestVersion(db)
```

Comment:
This latest version would keep changing as we restart right? Is there any concern ?

Reply:
Yea this will change to the migrate-height flag